website/docs/feature-flag-tutorials/serverless/implementing-feature-flags-in-aws-lambda.md

```diff
@@ -1,142 +1,174 @@
 ---
-title: How to implement feature flags in a serverless environment using AWS Lambda
+title: How to Implement Feature Flags in a Serverless Environment using AWS Lambda
+description: New to feature flags in serverless environments? This guide will walk you through a practical example using the Unleash Node.js SDK in AWS Lambda.
 slug: /feature-flag-tutorials/serverless/lambda
 ---
 
-For developers new to feature flags in serverless environments, this guide will walk you through practical examples using the [Unleash Node.js SDK](https://github.com/Unleash/unleash-client-node) in [AWS Lambda](https://aws.amazon.com/lambda/). These concepts can be easily adapted to other serverless solutions like [Google Cloud Functions](https://cloud.google.com/functions). Having feature flagging capabilities available inside of your serverless functions allows you to validate new parts of your serverless functions and stay in control of the feature exposure. 
+With feature flagging capabilities in your serverless functions, you can validate new parts of your serverless functions. This gives you complete control over feature exposure and simplifies serverless function versioning management.
 
+For developers new to feature flags in serverless environments, this guide will walk you through practical examples using the [Unleash Node.js SDK](https://github.com/Unleash/unleash-client-node) in [AWS Lambda](https://aws.amazon.com/lambda/). These concepts can be easily adapted to other serverless solutions like [Google Cloud Functions](https://cloud.google.com/functions).
+
+## Scenarios for AWS Lambda feature flags
+
+-   If you make a breaking change to a serverless function (e.g., new params), you usually need to create a new version with an appropriate URL routing scheme
+-   But sometimes you may need to add a non-breaking change, signature-wise, that may still have unwanted side effects for some consumers
+    -   Give a concrete example
```

Comment:
Is this section finished?  Looks like placeholder content still

Reply:
@ivarconr do we have an example for this? I'm assuming "signature-wise" is referring to code signing, but not sure.